Sprint-1/implement/sum.test.js

```diff
@@ -13,24 +13,49 @@ const sum = require("./sum.js");
 // Given an empty array
 // When passed to the sum function
 // Then it should return 0
-test.todo("given an empty array, returns 0")
+test ("Get an empty array, return 0", () =>{
+    EmptArr = sum([]);
```

Comment:
What is the scope of this variable (and the other ones you make in this test file)? Is it as narrow a scope as it could be?